graphql/logging.ts

```diff
@@ -59,5 +59,8 @@ export function logInContext(name: string, context: GraphQLContext) {
         sessionID = 'ADMIN';
     }
 
-    return getLogger(`${name} [${sessionID}]`);
+    const logger = getLogger(name);
+    logger.addContext('session', sessionID);
```

Comment:
This will make it easier to search for the category name & you are still able to filter by session ID